### ISSUE TITLE
The sitemap is generating an error in production

### ISSUE THREAD

**Issue by tomzx** [tomzx/blog.tomrochette.com#78]
It is apparently rendering an HTML page instead of the XML sitemap. I haven't been able to reproduce the issue in development yet.

```
This page contains the following errors:

error on line 40 at column 16: Opening and ending tag mismatch: meta line 0 and head
Below is a rendering of the page up to the first error.
```

**Comment by tomzx** [comment]
Apparently, the template is being overwritten by the `article.html.twig` template.

For now, I've temporarily fixed the issue by modifying the `sitemap.md` file with the following content:
```
---
template: sitemap
template_format: xml
---
```

**Comment by tomzx** [comment]
In the end, the issue was with the default template plugin, which override the template if none is set. I've implemented a small fix @ commit:141030a9a08768b83466c4d3f13c39e356394487.
